src/app/navbar/navbar.component.css

```diff
@@ -0,0 +1,128 @@
+:root{
+  font-size: 16px;
+  text-align:center;
+}
+
+.NavBar{
+
+  height: 6rem;
+  width: 100vw;
+  position: fixed;
+  flex-direction: row;
+  background-color: #315F42;
+  justify-content: flex-start;
+  transition: height 250ms ease;
+
+}
+
+.NavBar:hover{
+  height: 10rem;
+
+
+}
+
+.NavBar:hover .link-text{
+  display: flex;
+  flex-direction: column;
+
+
+}
+img{
+  display: flex;
+  justify-content: flex-start;
+
+}
+
+.nav-link i{
+
+  min-width: 2rem;
+  align-content: center;
+  align-items: center;
+  margin-left: 2rem;
+}
+.navbar-item li:hover{
+
+}
+.nav-link span{
+  margin-right: 4rem;
+}
+.nav-link {
+  text-align: center;
+align-content: center;
+  align-items: center;
+  height: 4rem;
+  text-decoration: none;
+  justify-content: center;
+display: flex;
+
+}
+.link-text{
+
+align-items: center;
+  display: none;
+  margin-top: 10rem;
+
+}
+.navbar-item{
+align-content: center;
+height: 100%;
+font-size:20px;
+}
+
+.navbar-item:last-child{
+  margin-left: auto;
+}
+.navigation {
+  font-family:Lato;
+  margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  list-style: none;
+  background: #315F42;
+  height: 6em;
+  text-align: center;
+
+}
+
+.navigation a {
+  text-decoration: none;
+  padding: 1em;
+  color: white;
+  display: flex;
+  align-items: center;
+}
+
+
+.navigation a:hover {
+  background: #5F314E;
+
+}
+
+
+body{
+  color: black;
+  background-color: white;
+  margin: 0;
+  padding:0;
+}
+main {
+  margin-top: 6rem;
+  padding:1rem ;
+}
+@media only screen and (min-width: 600px) {
+  .NavBar {
+    height: 6rem;
+    width: 100vw;
+  }
+}
+@media only screen and (max-width: 600px)
+{
+  .NavBar {
+    bottom: 0;
+    width: 100vw;
+    height: 4rem;
+  }
+
```

Comment:
Jeśli już używamy media queries to warto najpierw zastanowić się nad tym jakie chcemy mieć podejście do stosowania tego narzędzia w całej aplikacji?
- Możemy ustalić sobie swoje własne breakpointy (tak jak wyżej), co absolutnie nie jest błędem i być może sprawi, że aplikacja będzie maksymalnie dostosowana do naszych oczekiwań, jeśli chodzi o responsywność, jednak mieć na względzie, że takie podejście może mieć też swoje minusy i np. okazać się trudniejsze w utrzymaniu.
Można też skorzystać z zaproponowanych przez media query breakpointów wykorzystując wszystkie albo tylko niektóre z nich (w zależności od potrzeb) i mieć dzięki temu jakąś wspólną konwencję.
- Możemy używać zarówno min-width jak i max-width, jednak warto wiedzieć że takie rozwiązanie również może okazać się w dłuższej perspektywie trudniejsze w utrzymaniu. Możemy też skorzystać np. tylko z max-width i stylować poszczególne partie kodu kaskadowo.